packages/design-system-web/src/components/geov-entity-props-by-predicate/geov-entity-props-by-predicate.tsx

```diff
@@ -0,0 +1,163 @@
+import { Component, Host, h, Prop, State, EventEmitter, Event } from '@stencil/core';
+import { GeovPaginatorCustomEvent } from '../../components';
+import { sparqlJson } from '../../lib/sparqlJson';
+import { Binding } from '../geov-entity-properties/geov-entity-properties';
+import { PageEvent } from '../geov-paginator/geov-paginator';
+
+const qrOutgoingProps = (predicateId: string, objectId: string, pageSize: number, offset: number) => `
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX xml: <http://www.w3.org/XML/1998/namespace>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+PREFIX time: <http://www.w3.org/2006/time#>
+PREFIX ontome: <https://ontome.net/ontology/>
+PREFIX geov: <http://geovistory.org/resource/>
+
+SELECT DISTINCT ?subject ?subjectLabel ?subjectType ?subjectTypeLabel ?dt
+WHERE {
+  geov:${objectId} <${predicateId}> ?subject .
+  OPTIONAL {?subject rdfs:label ?subjectLabel .}
+  OPTIONAL {?subject rdf:type ?subjectType . OPTIONAL {?subjectType rdfs:label ?subjectTypeLabel .}}
+  BIND (datatype(?subject) AS ?dt) .
+}
+LIMIT ${pageSize} OFFSET ${offset}
+`;
+
+const qrIncomingProps = (predicateId: string, objectId: string, pageSize: number, offset: number) => `
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX xml: <http://www.w3.org/XML/1998/namespace>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+PREFIX time: <http://www.w3.org/2006/time#>
+PREFIX ontome: <https://ontome.net/ontology/>
+PREFIX geov: <http://geovistory.org/resource/>
+
+SELECT DISTINCT ?subject ?subjectLabel ?subjectType ?subjectTypeLabel ?dt
+WHERE {
+  ?subject <${predicateId}> geov:${objectId} .
+  OPTIONAL {?subject rdfs:label ?subjectLabel .}
+  OPTIONAL {?subject rdf:type ?subjectType . OPTIONAL {?subjectType rdfs:label ?subjectTypeLabel .}}
+  BIND (datatype(?subject) AS ?dt) .
+}
+LIMIT ${pageSize} OFFSET ${offset}
+`;
+
+@Component({
+  tag: 'geov-entity-props-by-predicate',
+  styleUrl: 'geov-entity-props-by-predicate.css',
+  shadow: true,
+})
+export class GeovEntityPropsByPredicate {
+  /**
+   * label
+   */
+  @Prop() entityId: string;
+  /**
+   * label
+   */
+  @Prop() props: Binding<string>;
```

Comment:
change this to `@Prop() predicate: string` and pass adequate string to it in `geov-entity-properties.tsx`